share/p2p/peers/options.go

```diff
@@ -19,8 +21,15 @@ type Parameters struct {
 
 	// EnableBlackListing turns on blacklisting for misbehaved peers
 	EnableBlackListing bool
+
+	// EnableShrexSub enables shrexsub for the peer manager. Setting it to false
+	// is used to enable shrex on bridge nodes without creating a second
+	// instance of shrexsub.
+	EnableShrexSub bool
```

Comment:
pls make it more clear that the way shrexsub is used in peerman is for listening, and that disabling it means node won't listen for recent block notifications